### PR TITLE
new(blog): add blog for Falco 0.39.0 

### PR DESCRIPTION
**Pull request by alacuku**
**What type of PR is this?**

> Uncomment one (or more) `/kind <>` lines:

> /kind bug

> /kind cleanup

> /kind design

> /kind user-interface

 /kind content

> /kind event

**Any specific area of the project related to this PR?**

> Uncomment one (or more) `/area <>` lines:

/area blog

> /area documentation

> /area community

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: content/en/blog/falco-0-39-0/index.md [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+
+* Prometheus Compliant metrics: some metrics have been reworked to follow the prometheus best practices [#3319](https://github.com/falcosecurity/falco/pull/3319);
+* Fixed ebpf drivers to use the correct memory barrier primitive for ARM64, preventing to read incomplete data from the ring buffers [#2067](https://github.com/falcosecurity/libs/pull/2067);
+* Fixed an issue where `stats messages` were written to `stdout` and could mix with regular Falco event output 
+  [#3338](https://github.com/falcosecurity/falco/pull/3338);
+* `fs.path` fields now account for `dirfd`, fixing discrepancies with `fd.name` [#1993](https://github.com/falcosecurity/libs/pull/1993);
+
+### Deprecations
+
+In Falco 0.40.0 we will remove the `--cri` and `--disable-cri-async` options.
```
Comment (leogr) [inline content/en/blog/falco-0-39-0/index.md:127]: It would be helpful to explain what to use instead.

**Review by leogr — COMMENTED** [review]
LGTM!

I just left a comment for a minor enhancement.

File: content/en/blog/falco-0-39-0/index.md (hunk 1 above)
Comment (alacuku) [reply content/en/blog/falco-0-39-0/index.md]: Done.

**Review by leogr — DISMISSED** [review]

**Review by LucaGuerra — APPROVED** [review]

**Pull request merged**